wiki/Beatmaps/fr.md

```diff
@@ -4,73 +4,79 @@ outdated: true
+Lorsqu'un beatmap est qualifié, il a passé le [ranking criteria](/wiki/ranking_criteria) et a été vérifié par deux [Beatmap Nominators](/wiki/Beatmap_Nominators). Les beatmaps qualifiés sont la dernière ligne de défense car ils doivent être dans ce statut pendant une semaine avant de devenir définitivement [ranked](#ranked) ou [approved](#approved).
 
-### Aimées
+Pendant la durée d'une semaine, cependant, [QATs](/wiki/QAT) sont autorisés à écrire des problèmes avec ces beatmaps. Si cela se produit, le statut du beatmapset redeviendra en attente et le processus recommencera à nouveau.
 
-Les beatmaps "aimées" ont l'icône de cœur (![Heart icon](/wiki/shared/status/loved.png)) dans l'écran de sélection des musiques.
+### Aimé
 
-Ces beatmaps peuvent ne pas remplir les [critères de classement](/wiki/ranking_criteria) mais sont _aimées_ de la communauté et sont donc souvent ajoutées suivant les votes de la communauté. Avant que la beatmap ne démarre, un dialogue s'affichera, informant que le statut de cette beatmap causera ce qui suit:
+*Pour l'histoire de la catégorie Aimé, voir : [History of Loved](/wiki/Beatmaps/History_of_Loved).*
 
-- Aucun [pp](/wiki/pp) ne sera accordé.
-- Tous les scores seront supprimés si le créateur met à jour la beatmap.
+Les cartes de battement aimées utilisent l'icône de coeur (![Heart icon](/wiki/shared/status/loved.png)) dans l'écran de sélection de la chanson.
 
-Ce statut permet à la beatmap d'avoir un tableau de classement.
+Ces beatmaps peuvent ne pas répondre aux [ranking criteria](/wiki/Ranking_Criteria), mais sont *aimés* par la communauté et sont donc régulièrement ajoutés en fonction des votes de la communauté afin qu'ils aient un tableau des leaders. Avant que le beatmap ne commence à jouer, un dialogue s'affiche, notant que
 
-Pour qu'une beatmap se voit accorder le statut d'"aimé", regardez le post de forum [Get your beatmap Loved!](https://osu.ppy.sh/community/forums/topics/549835).
+- Pas de [performance points (pp)](/wiki/Performance_Points) sera attribué.
+- Tous les scores seront supprimés si le créateur met à jour la carte des temps.
 
-### En Attente
+Pour qu'une beatmap soit aimée, elle doit passer par le [processus de vote communautaire](/wiki/Project_Loved#moving-a-beatmap-to-loved).
+
+
+### En attente
 
 _Voir aussi: [Procédure de Classement des Beatmaps](/wiki/Beatmap_ranking_procedure)_
 
-Les beatmaps en attente ont l'icône de point d'interrogation (![icône de point d'interrogation](/wiki/shared/status/pending.png)) dans l'écran de sélection des musiques.
 
-Une beatmap en attente attend généralement d'être [moddée](/wiki/modded). Les [créateurs](/wiki/Creators) doivent suivrent les [modders](/wiki/modders) de la communauté et attendre que deux [Nominateurs de Beatmaps](/wiki/Beatmap_Nominators) pour avoir leur beatmapset en dehors de ce statut.
+*Page principale : [Beatmap Ranking Procedure](/wiki/Beatmap_ranking_procedure)*
```

Comment:
this links to the same thing from above?

Reply:
Oh, it's obviously a mistake, I hadn't noticed, I'll take one off, thank you!